src/components/tag/Tag.props.ts

```diff
@@ -0,0 +1,28 @@
+import { TTagVariant, TTagSizeToken } from './Tag.types';
+
+type PTag = {
+    /**
+     * the variant of the tag for example, mentions, labels and shortcuts
+     */
+    variant: TTagVariant;
+    /**
+     * the text rendered inside the tag, for example @username, shortcut
+     */
+    text: string;
+    /**
+     * the size of the text rendered in the tag
+     */
+    size?: TTagSizeToken;
+    /**
+     * onClick handler
+     */
+    onClick?: () => void;
+    /**
+     * custom className
+     */
+    className?: string;
```

Comment:
we should always define `@default` values in JSDoc whenever we have them defined in the component as well, so we can make sure the docgen does catch it:

```suggestion
    /**
     * the variant of the tag for example, mentions, labels and shortcuts
     */
    variant: TTagVariant;
    /**
     * the text rendered inside the tag, for example @username, shortcut
     */
    text: string;
    /**
     * the size of the text rendered in the tag
     * @default 100
     */
    size?: TTagSizeToken;
    /**
     * onClick handler
     */
    onClick?: () => void;
    /**
     * custom className
     */
    className?: string;
```